html/Exercise 2/index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@100&display=swap" rel="stylesheet">
+  <title>Exercise 2</title>
+
+  <style>
+
+    * {
+      font-family: 'Raleway',sans-serif;
```

Comment:
check the design once. Entire design is so less in font weight that I can barely read anything
![Uploading Screenshot 2021-03-15 at 1.42.34 PM.png…]()